src/widgets/toolbar.rs

```diff
@@ -243,7 +243,15 @@ impl Default for Toolbar {
             hotkey: HotKey::new(None, "e"),
         };
 
-        Toolbar::new(vec![select, pen, knife, preview, rectangle, ellipse])
+        let measure = ToolbarItem {
+            name: "Measure",
+            icon: constrain_path(measure_path()),
+            hotkey: HotKey::new(None, "m"),
```

Comment:
it would be good to also add this to the tools menu in `src/menus.rs` (although I notice we're missing a few tools, there)

Reply:
Right, good call.

Reply:
The menu was updated in pr #188, so this is done.